### PR TITLE
feature: TR-2326. Display section titles

### PR DESCRIPTION
**Pull request by svoronuk**
Related to: https://oat-sa.atlassian.net/browse/TR-2326

AC1 and AC2 were implemented in initial "fizzy" layout. This task is more about AC3. For this AC, part of "fizzy" layout was reworked. Main difference in this layout - no splitting test items between test parts, test sections. Let say 'all-in-one'.

TODO:
- [x] add tests

How to test: see instructions in https://github.com/oat-sa/extension-tao-testqti/pull/2119

### CODE REVIEW

**Review by oatymart — COMMENTED** [review]
The change works well for me. Layout is correct with `displaySectionTitles` true or false.

- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [ ] Documentation is updated according to changes (if applicable)

If you have time, take a look at the test page http://127.0.0.1:xxxx/test/plugins/navigation/review/navigator/test.html when running `npm run test:keepAlive`. The CSS is kind of messed up there, maybe it can be fixed easily.

**Review by olga-kulish — APPROVED** [review]
Works well

File: src/plugins/navigation/review/navigator.js [hunk 1]
```diff
@@ -567,6 +568,9 @@ function navigatorFactory(config, map, context) {
     if (config.reviewLayout === 'fizzy') {
         navigatorTpl = fizzyTpl;
         navigatorTreeTpl = fizzyTreeTpl;
+        if(!config["displaySectionTitles"]){
```
[the remark this answers is not in the document]
Comment (svoronuk) [reply src/plugins/navigation/review/navigator.js]: Thx. Changed.

**Comment by svoronuk** [comment]
Update style for rendering container

**Review by oatymart — APPROVED** [review]
Changes look good.

**Pull request merged**